File: Cap 10/Topico 10.7/verdados/verifdados.swift
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 10 - Topico 10.7
 * 
 * Programa ..: OPERACAO COM SUBROTINA DE FUNCAO
 */

import Foundation

func igual(_ A: Int, _ B: Int) -> Bool {
    return A == B
}

var X: Int = 0
var Y: Int = 0

print("VERIFICACAO DE DADOS")
print("FUNCAO SIMPLES COM PASSAGEM DE PARAMETRO\n")

print("Informe o 1o. valor: ", terminator: "")
let inputX = readLine()
X = Int(inputX!) ?? 0

print("Informe o 2o. valor: ", terminator: "")
let inputY = readLine()
Y = Int(inputY!) ?? 0

print("")

if igual(X, Y) {
    print("Valores sao iguais")
} else {
    print("Valores sao diferentes")
}

File: Cap 10/Topico 10.7/verdados/verifdados.jl
#=
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 10 - Topico 10.7
 * 
 * Programa ..: OPERACAO COM SUBROTINA DE FUNCAO
=#

function IGUAL(A, B)
  return A == B
end

const X, Y = Ref{Int64}(0), Ref{Int64}(0)

println("VERIFICACAO DE DADOS")
println("FUNCAO SIMPLES COM PASSAGEM DE PARAMETRO")
println()
print("Informe o 1o. valor: ")
X[] = parse(Int64, readline())
print("Informe o 2o. valor: ")
Y[] = parse(Int64, readline())
println()
if IGUAL(X[], Y[])
  println("Valores sao iguais")
else
  println("Valores sao diferentes")
end
